app/src/parser.c

```diff
@@ -1630,6 +1630,69 @@ parser_error_t parser_getItemWithdrawRewardedDelegatedTokens(const parser_contex
     return PARSER_DISPLAY_IDX_OUT_OF_RANGE;
 }
 
+//SCO.11
+parser_error_t parser_getItemWithdrawUnstakedTokensSCO(const parser_context_t *ctx,
+                                       uint16_t displayIdx,
+                                       char *outKey, uint16_t outKeyLen,
+                                       char *outVal, uint16_t outValLen,
+                                       uint8_t pageIdx, uint8_t *pageCount) {
+    *pageCount = 1;
+    switch (displayIdx) {
+        case 0:
+            snprintf(outKey, outKeyLen, "Type");
+            snprintf(outVal, outValLen, "Withdraw Unstaked Tokens");
+            return PARSER_OK;
+        case 1:
+            snprintf(outKey, outKeyLen, "ChainID");
```

Comment:
```suggestion
            snprintf(outKey, outKeyLen, "Chain ID");
```

Reply:
All old transactions have ChainID string. Should I change the old ones, too?